code/game/objects/structures/girders.dm

```diff
@@ -338,32 +346,54 @@
 		playsound(src, 'sound/items/Welder.ogg', 100, 1)
 		if(do_after(user, 30, target = src))
 			user << "<span class='notice'>You slice apart the girder.</span>"
-			var/obj/effect/decal/remains/human/R = new (get_turf(src))
+			var/obj/item/stack/sheet/runed_metal/R = new(get_turf(src))
+			R.amount = 2
 			transfer_fingerprints_to(R)
 			qdel(src)
 
 	else if(istype(W, /obj/item/weapon/pickaxe/drill/jackhammer))
 		var/obj/item/weapon/pickaxe/drill/jackhammer/D = W
 		user << "<span class='notice'>Your jackhammer smashes through the girder!</span>"
-		var/obj/effect/decal/remains/human/R = new (get_turf(src))
+		var/obj/item/stack/sheet/runed_metal/R = new(get_turf(src))
+		R.amount = 2
 		transfer_fingerprints_to(R)
 		D.playDigSound()
 		qdel(src)
 
-/obj/structure/cultgirder/blob_act()
-	if(prob(40))
+	else if(istype(W, /obj/item/stack/sheet/runed_metal))
+		var/obj/item/stack/sheet/runed_metal/R = W
+		if(R.amount < 2)
+			user << "<span class='warning'>You need at least two sheets of runed metal to construct a runed wall!</span>"
+			return 0
+		user.visible_message("<span class='notice'>[user] begins laying runed metal on [src]...</span>", "<span class='notice'>You begin constructing a runed wall...</span>")
+		if(!do_after(user, 50, target = src))
+			return 0
+		user.visible_message("<span class='notice'>[user] plates [src] with runed metal.</span>", "<span class='notice'>You construct a runed wall.</span>")
+		R.amount -= 2
+		if(R.amount <= 0)
+			user.drop_item()
+			qdel(R)
+		new/turf/simulated/wall/cult(get_turf(src))
 		qdel(src)
 
+	else
+		..()
+
+/obj/structure/girder/cult/blob_act()
+	if(prob(40))
+		qdel(src)
 
-/obj/structure/cultgirder/ex_act(severity, target)
+/obj/structure/girder/cult/ex_act(severity, target)
 	switch(severity)
 		if(1)
 			qdel(src)
 		if(2)
 			if(prob(30))
-				new /obj/effect/decal/remains/human(loc)
+				var/obj/item/stack/sheet/runed_metal/R = new(get_turf(src))
+				R.amount = 2
 				qdel(src)
 		if(3)
 			if(prob(5))
-				new /obj/effect/decal/remains/human(loc)
+				var/obj/item/stack/sheet/runed_metal/R = new(get_turf(src))
+				R.amount = 2

```

Comment:
the new takes amount as an argument, so just do `new(get_turf(src), 2)` for all of these